src/features/people/PeoplePage/styled.js

```diff
@@ -2,10 +2,8 @@ import styled from "styled-components";
 import { Link } from "react-router-dom";
 
 export const Wrapper = styled.div`
-  max-width: 1368px;
+  max-width:${({ theme }) => theme.breakpoints.largest};
```

Comment:
Moze jest jakis useCase dla zmiany tego max-width to chetnie sie dowiem, ale tak jak pisalem wyzej wydaje mi sie, ze lepiej dac podstawowa wartosc jako 1368px bo tak jest w projekcie.

Reply:
To po prostu moja nadgorliwość, zrobię po staremu ;D